charts/kuberpult/values.yaml

```
@@ -78,7 +78,10 @@ cd:
     requests:
       cpu: 2
       memory: 3Gi
-  enableSqlite: true
+  db:
+    enabled: true
```

```suggestion
    enabled: false
```
database must be off by default for now.